testsuite/ui/webdriver.py

```diff
@@ -30,7 +30,7 @@ def install(self):
         if self.source == 'local':
             self.binary_path = ChromeDriverManager().install()
 
-    def start_session(self):
+    def start_browser(self):
```

Comment:
I would prefer to leave there session instead of the browser to avoid misunderstanding. Browser has webdriver and webdriver has browser means we browser.webdriver.browser. which is not a browser but webdriver session.

Reply:
The original was even more insane. Browser has session and webdriver. Webdriver had another session.

Reply:
The ideal name would be `selenium` as this is not session as well

Reply:
session/selenium is ok for me, it is up to you